File: Task-4/Fibonacci_Number.cpp
#include<bits/stdc++.h>
#include<iostream>
using namespace std;
// Write a function that will find the nth number in the Fibonacci Sequence
 int fib(int n) {
        int prev1=1;
        int prev2=0;
        if( n==0){
            return prev2;
        }
        for( int i=2; i<=n; i++){
            int curri=prev1+prev2;
            prev2=prev1;
            prev1=curri;
        }
        return prev1;
    }
    int main(){
        cout<<fib(20)<<endl;
    }

File: Task-4/Fibonacci_Number.java
//Program to find nth Fibonacci Number
public class Fibonacci_Number{
  public static void main(String[] args){
    int number = 8;
    System.out.println("Nth number in the series is: "+findFib(number));
  }
  
  public static int findFib(int number){
    if(number < 2)
      return number;
    int first = 0;
    int second = 1;
    int sum =0;
    for(int i =2; i< number + 1; i++){
      sum = first + second;
      first = second;
      second = sum;
    }
    return sum;
  }
}
    



//prints.  Nth number in the series is: 21
